pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1000,38 +1000,38 @@ func TestNodesWherePreemptionMightHelp(t *testing.T) {
 				"node2": framework.NewStatus(framework.UnschedulableAndUnresolvable, nodename.ErrReason),
 				"node3": framework.NewStatus(framework.UnschedulableAndUnresolvable, nodeunschedulable.ErrReasonUnschedulable),
 			},
-			expected: map[string]bool{"node1": true, "node4": true},
+			expected: map[string]bool{"node1": true},
```

Comment:
FWIW `node4` wasn't being returned without the change to `default_preemption.go` either. (Actually I think this test could be refactored to use a array `expected` instead of a map of bools; we should also check that the returned node list exactly matches the expected list.)